tests/integration/api/test_search_sort_desc.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import requests
+from .constants import API_URL
+from .utils import assert_response_status
+
+@pytest.mark.parametrize("endpoint, expected_status", [
+    ("/search?query=keyword%3ANASA&sort=date%3Adesc&filter=language%3ASerbian", 200),
```

Comment:
Recommend using `urllib.parse.quote` to escape the URL keys just to make it more readable.

Reply:
You can create a separate part of the tuple here for the language filter. Below you can then do 

```
assert item.get('language') == expected_language, f"Unexpected language: {item.get('language')}"
```